app/src/main/res/layout-sw600dp/profile_chooser_fragment.xml

```diff
@@ -57,7 +57,7 @@
           android:id="@+id/profile_select_text"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginStart="64dp"
+          android:layout_marginStart="@dimen/profile_chooser_fragment_layout_marginStart"
```

Comment:
@priyatanu dimension name should be `profile_chooser_fragment_layout_margin_start` instead.
And same goes for others.

Reply:
thank you for the review. I will update the files